installing/installing_bare_metal/bare-metal-using-bare-metal-as-a-service.adoc

```diff
@@ -0,0 +1,28 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="bare-metal-using-bare-metal-as-a-service"]
+= Using bare metal as a service
+include::_attributes/common-attributes.adoc[]
+:context: bare-metal-using-bmaas
+
+toc::[]
+
+{product-title}'s Bare Metal as a Service (BMaaS) feature enables you to provision and manage bare-metal nodes using the Metal^3^ API. These nodes, external to the {product-title} cluster, can host workloads requiring direct hardware access, such as legacy applications or high-performance computing tasks. BMaaS provides the following capabilities:
+
+* Provisioning of bare-metal nodes, including configuration and preparation for workloads.
```

Comment:
Suggestion: * Provisioning of bare-metal nodes, including initial configuration. (we want to make it clear we are not handling the workload part, our involvement ends with providing userData/networkData facilities)

Reply:
Fixed.